runconfig/config.go

```diff
@@ -17,6 +17,7 @@ type Entrypoint struct {
 	parts []string
 }
 
+// MarshalJSON Marshal (or serialize) the Entrypoint into the json format.
```

Comment:
I think it worth to mention that this method is needed to implement `json.Marshaller`. And I think this is main usage of this method.